### PR TITLE
Add support for CRI `verbose` fields

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
The remote runtime implementation now supports the `verbose` fields,
which are required for consumers like cri-tools to enable multi CRI
version support.
#### Which issue(s) this PR fixes:

Refers to https://github.com/kubernetes-sigs/cri-tools/pull/889

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
/assign
/triage accepted
/priority important-soon

(important soon as needed for cri tools)

File: pkg/kubelet/kuberuntime/kuberuntime_container.go [hunk 1]
```diff
@@ -505,12 +505,16 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
...
+		resp, err := m.runtimeService.ContainerStatus(c.Id, false)
 		if err != nil {
 			// Merely log this here; GetPodStatus will actually report the error out.
 			klog.V(4).InfoS("ContainerStatus return error", "containerID", c.Id, "err", err)
 			return nil, err
 		}
+		status := resp.GetStatus()
+		if status == nil {
+			return nil, errors.New("container status is nil")
+		}
```
Comment (pacoxu) [inline pkg/kubelet/kuberuntime/kuberuntime_container.go:517]: https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/errors.go#L28 

Nit: a constant like [ErrNetworkUnknown](https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/errors.go#L28 ) may be better

**Review by pacoxu — COMMENTED** [review]
LGTM except for the nit.

File: pkg/kubelet/kuberuntime/kuberuntime_container.go (hunk 1 above)
Comment (saschagrunert) [reply pkg/kubelet/kuberuntime/kuberuntime_container.go:518]: Fixed, added a new error to the `remote` package.

**Review by pacoxu — APPROVED** [review]
/lgtm

**Review by mrunalp — APPROVED** [review]

**Pull request merged**